### ISSUE TITLE
Migrations from docker container are are not terminating when complete

### ISSUE THREAD

**Issue by timorkal** [adonisjs/core#1249]
Hello I am having the same issue as here, when trying to run "node ace migration:run" or "adonis migration:run" from container to a Postgres DB.

https://github.com/adonisjs/core/issues/1027

I don't know why the issue was automatically closed, when there is a real issue going on.
Without this ability we can't do automatic migrations to our Postgres DB.

"adonis-version": "4.1.0"

Please help to figure out.

**Comment by McSneaky** [comment]
Can you provide git repo, where this issue is reproducible? Would make debugging issue a lot easier :slightly_smiling_face:

**Comment by timorkal** [comment]
It's a private repo for the company I work for, so I can't give access to it, but I can try to share some relevant parts of code.
I think this is a general issue so even the simplest project with connection to DB can be taken for debugging.

**Comment by ZaunSupremoXV** [comment]
I have the same error.
![Captura de Tela (21)](https://user-images.githubusercontent.com/67069017/87663725-aba57780-c73a-11ea-8fa0-9c4683fb3059.png)
![Captura de Tela (22)](https://user-images.githubusercontent.com/67069017/87663730-acd6a480-c73a-11ea-85fd-916b988d12b5.png)
![Captura de Tela (23)](https://user-images.githubusercontent.com/67069017/87663731-acd6a480-c73a-11ea-879d-0f5067ca3e96.png)
![Captura de Tela (24)](https://user-images.githubusercontent.com/67069017/87663733-ad6f3b00-c73a-11ea-92d1-ef2975a80e55.png)
![Captura de Tela (25)](https://user-images.githubusercontent.com/67069017/87663734-ad6f3b00-c73a-11ea-9c53-92689f728055.png)
![Captura de Tela (26)](https://user-images.githubusercontent.com/67069017/87663736-ae07d180-c73a-11ea-9ce1-328399bfaf5b.png)
![Captura de Tela (27)](https://user-images.githubusercontent.com/67069017/87663738-ae07d180-c73a-11ea-8256-1e7ee38611d6.png)


**Adonis version: 4.0.12
Node version: 12.18.1**

**Comment by thetutlage** [comment]
Closing in favor of discussion thread https://github.com/adonisjs/core/discussions/1251

**Comment by timorkal** [comment]
@thetutlage why did you close this issue, how @ZaunSupremoXV comment is related to my issue?
I am not having any issue with the migration itself, my issue that the migration is not exiting the session once it is finished, when running in a docker container.

**Comment by thetutlage** [comment]
First you have not provided much context on how to reproduce the issue or which version are you using?

2nd, we should only create issues for the confirmed bugs or improvements.

So it's better to discuss this in the discussions and please provide context to avoid back and forth

**Comment by timorkal** [comment]
@thetutlage 
1. I would not open a new issue if the first issue https://github.com/adonisjs/core/issues/1027 was not closed without any resolving and locked the ability to add comments.
2. I stated my adonisjs version - "4.1.0".
3. As I stated in my previous comment right after @McSneaky , It's a private repo so I can't share it, but I wrote there that I can supply parts of code, if you will direct me with which parts will be helpfull.
I also gave a simple example of how to reproduce it with a generic nodejs project that will run in docker and try to do a migration against PostgressDB. And that issue https://github.com/adonisjs/core/issues/1027 has a good example of how to do it.

Thank you

**Comment by thetutlage** [comment]
I replied to that issue https://github.com/adonisjs/core/issues/1027#issuecomment-520296737 and have not received any more info that can help me reproduce the issue.

You shared that you are using `Adonis 4.1.0`,I asked for the Lucid version, since the issue (if it is in Lucid).

If you cannot share the code or cannot help reproduce the issue, then I believe, there is no way for me to fix it.

**Comment by timorkal** [comment]
Sorry missed out that you need Lucid version.
Lucid - 6.1.3

I will try to give a better example in the next days, please just open the issue for now.

**Comment by timorkal** [comment]
Ok, after reviewing the issue again, it was a code problem (some jobs added in the migration phase) I had on my side which caused the migration not to exit the console.
